addon/components/o-s-s/attribute/country.ts

```diff
@@ -1,5 +1,5 @@
 import Component from '@glimmer/component';
-import { CountryData, countries } from '@upfluence/oss-components/utils/country-codes';
+import { type CountryData, countries } from '@upfluence/oss-components/utils/country-codes';
```

Comment:
🤔

Reply:
This is a result of https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax being enabled by default in TS5. 
IMO it's a good thing but as we were not using it before, it leads to a lot of files changed. But it can be disabled too